src/Neo.VM/JumpTable/JumpTable.Splice.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// JumpTable.Splice.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Neo.VM
+{
+    public partial class JumpTable
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public virtual void NewBuffer(ExecutionEngine engine, Instruction instruction)
+        {
+            int length = (int)engine.Pop().GetInteger();
+            engine.Limits.AssertMaxItemSize(length);
+            engine.Push(new Types.Buffer(length));
+        }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public virtual void Memcpy(ExecutionEngine engine, Instruction instruction)
+        {
+            int count = (int)engine.Pop().GetInteger();
```

Comment:
```suggestion
            int count = checked((int)engine.Pop().GetInteger());
```

Reply:
Shargon is right, we dont change any states of the vm in this pr. And i dont really think it a good idea to add checked. It indeed changes the behavior of contracts, need @superboyiii to check the state consistency carefully. The hardest part of the blockchain is you cant just update it at well even if there is a bug.

Reply:
@Jim8y you **can** change it. Just need a `fork`! If that was the case blockchain would go nowhere. They also said blockchain can't work either, lots of singularities; well that was all solved when bitcoin was created; so I think we can solve our sample issues with a fork... Lets not forget `The Last Resort`, create another chain 🤣.